src/main/webapp/mocks/server-handlers.js

```diff
@@ -0,0 +1,13 @@
+// src/mocks/serverHandlers.js
+import { rest } from 'msw';
+import { mockServerResponse } from './server-response';
+
+export const serverHandlers = [
+    rest.get('/servicesNS/nobody/-/mockGeneratedEndPointUrl', (req, res, ctx) =>
+        res(ctx.json(mockServerResponse))
+    ),
+    rest.get('/servicesNS/nobody/-/splunk_ta_aws_aws_all_accounts', (req, res, ctx) =>
```

Comment:
Looks like very specific path (Calling something from AWS TA). If any of the tests calls this endpoint - I would consider changing to some generic path in test. If it's in the component code - it looks verys suspicious - what is the point in creating components that work with one specific TA?

It does not have to be fixed now - feel free to create separate Jira for that issue

Reply:
Agree. Fixed